examples/templateservicebroker/templateservicebroker-template.yaml

```diff
@@ -138,3 +139,57 @@ objects:
   - kind: ServiceAccount
     namespace: ${NAMESPACE}
     name: apiserver
+
+# for the SA for the service catalog to use when calling the tsb
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    namespace: ${NAMESPACE}
+    name: templateservicebroker-client
+
+# allow the SC's SA to call the TSB
+- kind: ClusterRoleBinding
+  apiVersion: v1
+  metadata:
+    name: templateservicebroker-client
+  roleRef:
+    name: system:openshift:templateservicebroker-client
+  userNames:
+  - system:serviceaccount:${NAMESPACE}:templateservicebroker-client
+
+# copy of the SC's SA's token secret which will be retrieved by the SC for use 
+# when calling the TSB
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    namespace: ${NAMESPACE}
+    name: templateservicebroker-client
+    annotations:
+      kubernetes.io/service-account.name: templateservicebroker-client
+  type: kubernetes.io/service-account-token
+
+# allow the SC to read the SA's secret defined above
+- kind: Role
+  apiVersion: v1
+  metadata:
+    name: templateservicebroker-auth-reader
+    namespace: ${NAMESPACE}
+  rules:
+  - apiGroups:
+    - ""
+    resourceNames:
+    - templateservicebroker-client
+    resources:
+    - secrets
+    verbs:
+    - get
+- kind: RoleBinding
+  apiVersion: v1
+  metadata:
+    namespace: ${NAMESPACE}
+    name: templateservicebroker-auth-reader
+  roleRef:
+    namespace: ${NAMESPACE}
+    name: templateservicebroker-auth-reader
+  userNames:
+    - system:serviceaccount:service-catalog:service-catalog-controller
```

Comment:
This namespace isn't correct.  Needs to be either `kube-something` or `openshift-something`

Reply:
` Error: project.project.openshift.io "kube-service-catalog" is forbidden: cannot request a project starting with "kube-"`

Reply:
Create a namespace, not a project.

Reply:
thanks @enj